ci/ReleaseTool/ReleaseCommand.cs

```diff
@@ -389,7 +245,7 @@ private Release CreateRelease(GitHubClient gitHubClient, Repository gitHubRepo,
 ## Upgrading
 
 * You can find the corresponding UnrealEngine version(s) [here](https://github.com/improbableio/UnrealEngine/releases).
-* You can find the corresponding UnrealGDKExampleProject version [here](https://github.com/spatialos/UnrealGDKExampleProject/releases).
+* You can find the corresponding UnrealGDKExampleProject version [here](https://github.com/spatialos/UnrealGDKExampleProject/releases/tag/{options.Version}).
```

Comment:
Nice

Reply:
I could also try to link to the engine versions dynamically (e.g. do an entry for each version)

Reply:
This is good